.nais/ebms-provider-dev.yaml

```diff
@@ -72,6 +74,8 @@ spec:
       value: "4"
     - name: MAX_CONNECTION_POOL_SIZE_FOR_ADMIN
       value: "1"
+    - name: ASYNC_RECEIVER
+      value: "true"
```

Comment:
Lagrer vi en string her? (Spør i tilfelle det ikke var med vilje, men kanskje det parses som boolean uansett?)

Reply:
NAIS [krever](https://doc.nais.io/workloads/application/reference/application-spec/#envvalue) at både int og bool er skrevet på denne måten. Litt opp til hvordan man leser det inn i applikasjonen for å sikre at det blir riktig. I dette tilfellet leser jeg det bare inn som en String, og bruker kotlin sin `.toBoolean()`. Den er relativt eksplisitt på den måten at kun "true" (case insensitive) blir `true`. Alt annet gir `false`, inkludert null.